superduperdb/backends/sqlalchemy/metadata.py

```diff
@@ -18,6 +21,15 @@ def as_dict(self):
         return {c.name: getattr(self, c.name) for c in self.__table__.columns}
 
 
+class QueryID(Base):  # type: ignore[valid-type, misc]
+    __tablename__ = 'query_id_table'
+
+    query_id = Column(Integer, primary_key=True, autoincrement=False)
+    query = Column(String)
```

Comment:
`JSON`?

Reply:
hmm will JSON be supported by all backend?

Reply:
It's supported by `SQLAlchemy`.

Reply:
ok done